pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -310,6 +311,63 @@ func newOpenStack(cfg Config) (*OpenStack, error) {
 	return &os, nil
 }
 
+func configFromEnv() (cfg Config, ok bool) {
```

Comment:
`cfg` must be a pointer since it's assigned to. It might be more "obvious" or readable if this func became a method instead?